source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -485,7 +485,9 @@ def test_journey_with_external_uri_from_bragi(self):
             'https://host_of_bragi/autocomplete?q=bobette&{p}'.format(p=params): (bragi_bobette, 200),
             'https://host_of_bragi/features/bobette?pt_dataset=main_routing_test': (bragi_bobette, 200),
             'https://host_of_bragi/autocomplete?q=20+rue+bob&{p}'.format(p=params): (bob_street, 200),
-            'https://host_of_bragi/features/{}?pt_dataset=main_routing_test'.format(check_utils.r_coord): (bob_street, 200)
+            'https://host_of_bragi/reverse?lat={lat}&lon={lon}&pt_dataset=main_routing_test'
```

Comment:
Maybe that will be the same interface for place nearby, giving a limit and some types. In this case, we care.

I can remove it if you want, or just let it like that by default as it's less work without identified problems.